events/schema.py

```diff
@@ -400,6 +401,9 @@ class Meta:
 
 
 class EnrolmentNode(DjangoObjectType):
+
+    reference_id = graphene.String(description="An unique encoded reference id")
+
```

Comment:
Offering the enrolment reference id to authorized guardians, so the UI can recreate the qrcode.